### PR TITLE
refactor(country-of-birth): [EMU-3468] update country list to iso 3166 1 standard

### PR DESCRIPTION
**Pull request by talo242**

### CODE REVIEW

File: src/country.ts [hunk 1]
```diff
@@ -568,42 +656,52 @@ export const allCountries: Country[] = [
...
-  { name: 'South Sudan, Republic of', code: 'SS' },
   { name: 'Suriname', code: 'SR' },
-  { name: 'Eswatini', code: 'SZ' },
+  { name: 'Svalbard and Jan Mayen', code: 'SJ' },
+  { name: 'Swaziland', code: 'SZ' },
   { name: 'Sweden', code: 'SE' },
   { name: 'Switzerland', code: 'CH' },
   { name: 'Syrian Arab Republic', code: 'SY' },
-  { name: 'Taiwan', code: 'TW' },
+  { name: 'Taiwan, Province of China', code: 'TW' },
```
Comment (vincentaudoire) [inline src/country.ts:676]: Leave it as Taiwan for now 😅

File: src/country.ts [hunk 2]
```diff
@@ -568,42 +656,52 @@ export const allCountries: Country[] = [
...
   { name: 'United Arab Emirates', code: 'AE' },
   { name: 'United Kingdom', code: 'GB' },
   { name: 'United States', code: 'US' },
+  { name: 'United States Minor Outlying Islands', code: 'UM' },
   { name: 'Uruguay', code: 'UY' },
   { name: 'Uzbekistan', code: 'UZ' },
   { name: 'Vanuatu', code: 'VU' },
-  { name: 'Venezuela', code: 'VE' },
-  { name: 'Vietnam', code: 'VN' },
+  { name: 'Venezuela, Bolivarian Republic of', code: 'VE' },
```
Comment (vincentaudoire) [inline src/country.ts:699]: Leave it to Venezuela?

File: src/country.ts [hunk 3]
```diff
@@ -512,52 +586,66 @@ export const allCountries: Country[] = [
   { name: 'Liechtenstein', code: 'LI' },
   { name: 'Lithuania', code: 'LT' },
   { name: 'Luxembourg', code: 'LU' },
-  { name: 'North Macedonia, Republic of', code: 'MK' },
+  { name: 'Macao', code: 'MO' },
+  { name: 'Macedonia, the Former Yugoslav Republic of', code: 'MK' },
```
Comment (vincentaudoire) [inline src/country.ts:590]: I think they renamed to "North Macedonia"

File: src/country.ts [hunk 4]
```diff
@@ -427,11 +480,15 @@ export const allCountries: Country[] = [
   { name: 'Belgium', code: 'BE' },
   { name: 'Belize', code: 'BZ' },
   { name: 'Benin', code: 'BJ' },
+  { name: 'Bermuda', code: 'BM' },
   { name: 'Bhutan', code: 'BT' },
-  { name: 'Bolivia', code: 'BO' },
+  { name: 'Bolivia, Plurinational State of', code: 'BO' },
```
Comment (vincentaudoire) [inline src/country.ts:485]: Could leave as Bolivia I believe?

File: src/country.ts (hunk 1 above)
Comment (talo242) [reply src/country.ts:676]: That's exactly what I thought 😅, geopolitics are everywhere hahaha

**Comment by talo242** [comment]
After a second review, I think I'll move this PR as a draft. There are a couple of flags that we are missing and I think it'll be better to first make the changes so the applications are not rejected by TK's API and then we can update this list.

**Review by kseha1080 — COMMENTED** [review]
Is there a reason why we're adding these countries again? We decided earlier this year to take out countries that are "territories" of larger ones - Guam, Macao, French Polynesia, etc. to name a few.

**Comment by vincentaudoire** [comment]
@talo242 I agree with @kseha1080, I think we purposely removed territories to simplify and have countries instead

**Comment by talo242** [comment]
@vincentaudoire @kseha1080 I'll close this PR because I think the list of countries we have right now works fine.

**Pull request closed without merging**